artfinder/django-js-reverse
# Add ability to specify names for include and exclude

**What I Did**

- Added functionality to include and exclude by URL name, to compliment the existing URL namespace functionality

**How I Did It**

- Added new config variables for excluding names and including names
- Refactored the include/exclude functionality to make it easier to understand (although less performant)
- Added new tests for the url name inclusions / exclusions

**How to Verify It**
- check out https://github.com/artfinder/world/pull/5244
- do a pip install
- run `bauhaus/manage.py collectstatic_js_reverse && cat /tmp/af-static/django_js_reverse/js/reverse.js  | less`
- check that the daily-finds url is present
- check that the checkout:confirmation url is not present
